src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -944,6 +944,8 @@ ConnectionHandlerImpl::TransportTypeProfileStringFromDeviceHandle(
     return std::string("IAP_CARPLAY");
   } else if (connection_type == "CLOUD_WEBSOCKET") {
     return std::string("WEBSOCKET");
+  } else if (connection_type == "WEBENGINE_WEBSOCKET") {
+    return std::string("WEBSOCKET");
```

Comment:
Seems these values are also missing from the related description in the INI

Reply:
I also think it might make sense to give this a different type name from `CLOUD_WEBSOCKET`, since all of the other types have a unique name